test/job.coffee

```diff
@@ -66,4 +67,10 @@ describe "./#{pkg.name} job", () ->
     job jobId, command, (err) ->
       expect(datalink.jobStatus).to.be.calledOnce
       expect(datalink.jobStatus).to.be.calledWith jobId
-      cb err
\ No newline at end of file
+      cb err
+
+  it 'should print the current job status when called without an id.', (cb) ->
+    job null, command, (err) ->
+      expect(datalink.jobStatus).to.be.calledOnce
+      expect(datalink.jobStatus).to.be.calledWith null
```

Comment:
shouldn't this be 
`expect(datalink.jobStatus).to.be.calledWith project.lastJobId` ?